### PR TITLE
WT-6081 Switch wtperf table config defaults closer to MongoDB's config

### PR DESCRIPTION
**Pull request by sulabhM**

### CODE REVIEW

**Comment by sulabhM** [comment]
The python test failure is not related to this change request. There is already a ticket for that test failure

**Comment by sulabhM** [comment]
Also, note I have created WT-6084, because the 3 workloads I changed were failing on develop before my change and continue to do so after my change.

Irrespective of the existing failure, I think they were type=lsm before but suit to type=file better being btree workloads.

File: bench/wtperf/wtperf_opt.i [hunk 1]
```diff
@@ -171,9 +171,8 @@ DEF_OPT_AS_UINT32(scan_table_count, 0,
 DEF_OPT_AS_CONFIG_STRING(sess_config, "", "session configuration string")
 DEF_OPT_AS_UINT32(session_count_idle, 0, "number of idle sessions to create. Default 0.")
 DEF_OPT_AS_CONFIG_STRING(table_config,
-  "key_format=S,value_format=S,type=lsm,exclusive=true,"
```
Comment (agorrod) [inline bench/wtperf/wtperf_opt.i:174]: Please add a comment here explaining that the defaults match the values used by MongoDB for collections.

Comment (sulabhM) [reply bench/wtperf/wtperf_opt.i:174]: done

Comment (sueloverso) [reply bench/wtperf/wtperf_opt.i:174]: If you changed the default here, then why did you need/want to add in all the `table_config` calls to the 500m files? Wouldn't this change alone be sufficient?

Comment (sulabhM) [reply bench/wtperf/wtperf_opt.i:174]: The default "type" changed from lsm to file. These three 500m configurations do not set a type. I thought "type=file" suits these configs, but I could be wrong. These configs failed to run with or without my changes, WT-6084 would get into fixing them.

**Pull request merged**